### PR TITLE
Speed up `scripts/format-stylish.sh`

### PR DESCRIPTION
**Pull request by wenkokke**
# Description

- Set `exit_code` to `error_on_format`.
- Batch formatting with 50 files at a time.

### CODE REVIEW

File: scripts/format-stylish.sh [hunk 1]
```diff
@@ -18,6 +18,6 @@ fi
 # Check Haskell files with stylish-haskell
 echo "Formatting Haskell source files with stylish-haskell version ${stylish_haskell_required_version}"
 # shellcheck disable=SC2016
-if ! git ls-files --exclude-standard --no-deleted --deduplicate '*.hs' | xargs -L 1 sh -c 'echo "$0" && stylish-haskell -i -c .stylish-haskell.yaml "$0"'; then
+if ! git ls-files --exclude-standard --no-deleted --deduplicate '*.hs' | xargs -L 50 stylish-haskell -i -c .stylish-haskell.yaml; then
```
Comment (jorisdral) [inline scripts/format-stylish.sh:21]: Does batching make a difference?

File: scripts/pre-commit.sh [hunk 2]
```diff
@@ -14,14 +14,14 @@ if [ ! "${unstaged_haskell_files}" = "" ]; then
...
-    echo "Found unstaged Cabal files"
-    echo "${unstaged_cabal_files}"
-    exit 1
-fi
+# unstaged_cabal_files="$(git ls-files --exclude-standard --no-deleted --deduplicate --modified '*.cabal')"
+# if [ ! "${unstaged_cabal_files}" = "" ]; then
+#     echo "Found unstaged Cabal files"
+#     echo "${unstaged_cabal_files}"
+#     exit 1
+# fi
```
Comment (jorisdral) [inline scripts/pre-commit.sh:22]: Why is this commented out now?

**Review by jorisdral — APPROVED** [review]

File: scripts/pre-commit.sh (hunk 2 above)
Comment (mheinzel) [inline scripts/pre-commit.sh:22]: What's the reason for commenting this out?

Comment (mheinzel) [reply scripts/pre-commit.sh:22]: Ah, already asked :smile:

Comment (wenkokke) [reply scripts/pre-commit.sh:22]: Oops!

File: scripts/format-stylish.sh (hunk 1 above)
Comment (wenkokke) [reply scripts/format-stylish.sh:21]: Yes. Massive. In fact, it's the only thing that made a difference.

File: scripts/pre-commit.sh (hunk 2 above)
Comment (wenkokke) [reply scripts/pre-commit.sh:22]: Oops!

**Comment by jorisdral** [comment]
Could you squash the commits before I press the merge button?

**Comment by wenkokke** [comment]
Closed in favour of #465.

**Pull request closed without merging**